packages/office-ui-fabric-react/src/components/SelectedItemsList/SelectedPeopleList/Items/EditingItem.tsx

```diff
@@ -54,7 +54,7 @@ export class EditingItem extends BaseComponent<IEditingSelectedPeopleItemProps,
           onBlur={this._onInputBlur}
           onClick={this._onInputClick}
           data-lpignore={true}
-          className={styles.editingInput}
+          className={css('ms-BasePicker-input', styles.editingInput)}
```

Comment:
does this need to be `ms-BasePicker-input`? Can we use `ms-EditingItem-input` instead to avoid any unexpected collisions?